cmd/contour/shutdownmanager.go

```diff
@@ -82,13 +85,17 @@ func newShutdownManagerContext() *shutdownmanagerContext {
 	}
 }
 
-func newShutdownContext() *shutdownContext {
+func newShutdownContext(adminPort int) *shutdownContext {
```

Comment:
I'd call this `defaultAdminPort`, as ideally it should be setting the default, that can be overridden by a flag in the outer `registerShutdown` function.